host-configs/llnl/blueos_3_ppc64le_ib_p9/clang@upstream_link_with_nvcc.cmake

```diff
@@ -58,15 +58,16 @@ set(BLT_OPENMP_LINK_FLAGS "-Xlinker -rpath -Xlinker ${OMP_HOME}/lib -L${OMP_HOME
 #------------------------------------------------------------------------------
 set(ENABLE_CUDA ON CACHE BOOL "")
 
-set(CUDA_TOOLKIT_ROOT_DIR "/usr/tce/packages/cuda/cuda-10.1.168" CACHE PATH "")
+set(CUDA_TOOLKIT_ROOT_DIR "/usr/tce/packages/cuda/cuda-11.1.1" CACHE PATH "")
 set(CMAKE_CUDA_COMPILER "${CUDA_TOOLKIT_ROOT_DIR}/bin/nvcc" CACHE PATH "")
 set(CMAKE_CUDA_HOST_COMPILER ${CMAKE_CXX_COMPILER} CACHE PATH "")
 
-set (_cuda_arch "sm_70")
-set (CMAKE_CUDA_FLAGS "-restrict -arch ${_cuda_arch} -std=c++11 --expt-extended-lambda -G" CACHE STRING "" )
+set(CMAKE_CUDA_ARCHITECTURES "70" CACHE STRING "")
+set(_cuda_arch "sm_${CMAKE_CUDA_ARCHITECTURES}")
```

Comment:
Does this flag need to be set if CMAKE_CUDA_ARCHITECTURES is set?

Reply:
Maybe a legacy of FindCUDA... todo when we revamp our CUDA logic

Reply:
Should we add it to an issue? (or start one)?